website/content/en/v0.28/troubleshooting.md

```diff
@@ -314,6 +314,35 @@ When a node is launched by Karpenter, it is assigned to a subnet within your VPC
 
 For more troubleshooting information on why your pod may have a `FailedCreateSandbox` error, view the [EKS CreatePodSandbox Knowledge Center Post](https://repost.aws/knowledge-center/eks-failed-create-pod-sandbox).
 
+#### Windows pods are failing to launch
+
+The following solution(s) may resolve your issue if you are seeing an error similar to the following when attempting to launch Windows pods.
+
+```
+Failed to create pod sandbox: rpc error: code = Unknown desc = failed to setup network for sandbox "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx": plugin type="vpc-bridge" name="vpc" failed (add): failed to parse Kubernetes args: pod does not have label vpc.amazonaws.com/PrivateIPv4Address
+```
+
+This error typically occurs if you have not enabled Windows support. If you are provisioning Windows nodes, you need to [enable Windows support](https://docs.aws.amazon.com/eks/latest/userguide/windows-support.html#enable-windows-support) for your cluster by enabling Windows IPAM.
+
+##### Solutions
+
+1. Set `enable-windows-ipam` to true in the `amazon-vpc-cni` config map. See documentation [Enabling Windows support](https://docs.aws.amazon.com/eks/latest/userguide/windows-support.html#enable-windows-support) for more details regarding how this can be accomplished.
```

Comment:
Same as comment from earlier about redirecting to EKS docs about all the info regarding enabling Windows support. We should not document the specific things to do since they are just a part of a wider set of steps.

Reply:
Ack